src/DynamoCoreWpf/Controls/HeaderStrip.xaml

```diff
@@ -0,0 +1,114 @@
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:controls="clr-namespace:Dynamo.Controls"
+             mc:Ignorable="d"
+             d:DesignHeight="300"
+             d:DesignWidth="300"
+             Name="HeaderStripControl">
+    <UserControl.Resources>
+        <controls:SelectedItemToActiveConverter x:Key="SelectedItemToActiveConverter"
+                                                NormalColor="Transparent"
+                                                ActiveColor="#393939" />
+
+        <Style x:Key="HeaderTextBlockStyle"
+               TargetType="{x:Type TextBlock}">
+            <Setter Property="VerticalAlignment"
+                    Value="Center" />
+            <Setter Property="Foreground"
+                    Value="#999999" />
+            <Setter Property="FontSize"
+                    Value="9" />
+        </Style>
+        <Style x:Key="HeaderListButtonStyle"
+               TargetType="{x:Type Button}">
+            <Setter Property="Template">
+                <Setter.Value>
+                    <ControlTemplate>
+                        <Border x:Name="container"
+                                Padding="8,3,8,3"
+                                BorderThickness="1"
+                                BorderBrush="#444444"
+                                CornerRadius="2"
+                                Background="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=Background}">
+                            <TextBlock x:Name="text"
+                                       Style="{StaticResource HeaderTextBlockStyle}"
+                                       Text="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=Content}">
+                            </TextBlock>
+                        </Border>
+                        <ControlTemplate.Triggers>
+                            <Trigger Property="Button.IsMouseOver"
+                                     Value="True">
+                                <Setter TargetName="container"
+                                        Property="Background"
+                                        Value="#393939" />
+                                <Setter TargetName="text"
+                                        Property="Foreground"
+                                        Value="#AAAAAA" />
+                            </Trigger>
+                            <Trigger Property="Button.IsPressed"
+                                     Value="True">
+                                <Setter TargetName="container"
+                                        Property="Background"
+                                        Value="#222222" />
+                            </Trigger>
+                        </ControlTemplate.Triggers>
+                    </ControlTemplate>
+                </Setter.Value>
+            </Setter>
+        </Style>
+
+        <DataTemplate x:Key="OneHeaderTemplate">
```

Comment:
We can rename ```OneHeaderTemplate``` to ```SingleItemListTemplate``` so that its intents are clearer. At the same time I'm also wondering, why don't we just merge ```SingleItemListTemplate``` with ```HeadersListTemplate```? I think we can have an one-item list box, right? I'm not entirely sure if we should use a template selector for this (can we not make it just a list with a single item?)